include/modem/lte_lc.h

```diff
@@ -651,6 +651,47 @@ enum lte_lc_reduced_mobility_mode {
 	LTE_LC_REDUCED_MOBILITY_DISABLED = 2,
 };
 
+/**
+ * UICC power saving mode.
+ */
+enum lte_lc_uiccpowersave_mode {
+	/**
+	 * Default functionality.
+	 * The UICC is deactivated during eDRX only when:
+	 * - \li It is allowed by the EF-AD (Elementary File - Administrative Data)
+	 * - \li Application PIN is disabled
+	 * - \li Sleep mode time is longer than the minimum time limit (1 minute in ultra
+	 *   low power mode and 5 minutes in other power modes)
+	 *
+	 * @note AT&T has a shorter minimum time of 20.48 seconds.
+	 *
+	 * Proactive polling interval can be modified during eDRX sleep only if allowed in EF-AD.
+	 */
+	LTE_LC_UICCPOWERSAVE_DEFAULT = 0,
+
+	/**
+	 * If UICC deactivation is not allowed due to some conditions mentioned in
+	 * @ref LTE_LC_UICCPOWERSAVE_DEFAULT, the proactive polling interval is increased to
+	 * match the eDRX sleep time regardless of EF-AD content.
+	 * If the interval is already longer than the sleep time, it is not affected.
+	 */
+	LTE_LC_UICCPOWERSAVE_POLLING = 1,
+
+	/**
+	 * Bypass conditions related to EF-AD, PIN state, and minimum deactivation time when
+	 * deactivating the UICC.
+	 * After deactivation on eDRX sleep, the UICC is activated only when the EMM (EPS
+	 * Mobility Management) is starting to initiate signalling to the network.
+	 */
+	LTE_LC_UICCPOWERSAVE_BYPASS = 2,
```

Comment:
I tried that in the first implementation, but it wouldn't be as intuitive to remember for the user.

Reply:
Neither solution being perfect for default, I think I'd vote for consistency and have `MODE_0`. Just because `MODE_DEFAULT` is not some well-known default functionality that people could understand without reading the description.
Also, normally the default value is just one of the values which doesn't have wording "default" in it although we do have those "default" ones too.

Reply:
Discussed this with @tokangas. Because `MODE_0` is default functionality of the modem, I'm open for `MODE_DEFAULT`. I had somehow assumed that there would be some other "no power saving state" if this AT command is not used.